ui/pages/confirmations/confirm/__snapshots__/confirm.test.tsx.snap

```diff
@@ -1779,53 +1653,13 @@ exports[`Confirm should match snapshot signature - typed sign - order 1`] = `
             class="mm-box mm-box--margin-top-2 mm-box--display-flex"
           >
             <div
-              class=""
-            >
-              <div
-                class="identicon"
-                style="height: 32px; width: 32px; border-radius: 16px;"
-              >
-                <div
-                  style="border-radius: 50px; overflow: hidden; padding: 0px; margin: 0px; width: 32px; height: 32px; display: inline-block; background: rgb(200, 20, 59);"
-                >
-                  <svg
-                    height="32"
-                    width="32"
-                    x="0"
-                    y="0"
-                  >
-                    <rect
-                      fill="#017B8E"
-                      height="32"
-                      transform="translate(0.7994460693567097 -0.8216011156969824) rotate(317.9 16 16)"
-                      width="32"
-                      x="0"
-                      y="0"
-                    />
-                    <rect
-                      fill="#F2B602"
-                      height="32"
-                      transform="translate(-7.493396036912253 11.409935428284795) rotate(134.1 16 16)"
-                      width="32"
-                      x="0"
-                      y="0"
-                    />
-                    <rect
-                      fill="#FA8E00"
-                      height="32"
-                      transform="translate(-25.712272690754798 -11.558881043268883) rotate(364.0 16 16)"
-                      width="32"
-                      x="0"
-                      y="0"
-                    />
-                  </svg>
-                </div>
-              </div>
-            </div>
+              class="identicon__image-border"
+              style="height: 32px; width: 32px; border-radius: 16px;"
+            />
             <div
               class="mm-box mm-text mm-avatar-base mm-avatar-base--size-xs mm-avatar-network confirm_header__avatar-network mm-text--body-xs mm-text--text-transform-uppercase mm-box--display-flex mm-box--justify-content-center mm-box--align-items-center mm-box--color-text-default mm-box--background-color-background-alternative mm-box--rounded-full mm-box--border-color-transparent box--border-style-solid box--border-width-1"
             >
-              C
+              ?
```

Comment:
Looks like this is a test regression on ConfirmPage while adding support in the Header test. Could we continue to support the network in this test?

Reply:
Looks like this file is no longer part of this PR.